database/migrations/2021_03_06_182839_create_booking_statuses_table.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateBookingStatusesTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('booking_statuses', function (Blueprint $table) {
+            $table->id();
+            $table->string('label');
```

Comment:
If we're going to use the statuses package, shouldn't we have a foreignId reference?

Reply:
Please dont use status package for now - use BookingStatus model please.

Reply:
Pending confirmation from https://github.com/tipoff/bookings/issues/49

Reply:
Using Status package. Converted this migration to add statuses intended for bookings.